### ISSUE TITLE
dataset initialization error

### ISSUE THREAD

**Issue by xiaotubaobiu** [10Ring/LAA-Net#11]
train_dataloader = DataLoader(train_dataset, 
                                          batch_size=cfg.TRAIN.batch_size * len(cfg.TRAIN.gpus),
                                          shuffle=True,
                                          pin_memory=cfg.DATASET.PIN_MEMORY,
                                          num_workers=cfg.DATASET.NUM_WORKERS,
                                          worker_init_fn=train_dataset.train_worker_init_fn,
                                          collate_fn=train_dataset.train_collate_fn)
        
  There is no such attribute（    train_dataset.train_worker_init_fn） when the dataset is initialized. What is the meaning of this line of code?

**Comment by Swilder7** [comment]
I have met the same problem. Could you please tell me how you solved it?
